test/e2e/framework/deployment/fixtures.go

```diff
@@ -72,7 +73,8 @@ func NewDeployment(deploymentName string, replicas int32, podLabels map[string]s
 
 // CreateDeployment creates a deployment.
 func CreateDeployment(ctx context.Context, client clientset.Interface, replicas int32, podLabels map[string]string, nodeSelector map[string]string, namespace string, pvclaims []*v1.PersistentVolumeClaim, command string) (*appsv1.Deployment, error) {
-	deploymentSpec := testDeployment(replicas, podLabels, nodeSelector, namespace, pvclaims, false, command)
+	// TODO: let the caller decide about the security level.
```

Comment:
mark the todo with a name or an issue

Reply:
I started writing an issue and then noticed that the function is only called in three places. Addressing the TODO by passing admissionapi.LevelRestricted is now in this PR.